Makefile

```diff
@@ -89,6 +92,8 @@ release-image.arm64v8: clean
 	REGISTRY=$(RELEASE_REGISTRY) \
 	IMAGE=$(RELEASE_IMAGE)-arm64 \
 	CONTROLLER_IMAGE=$(RELEASE_CONTROLLER_IMAGE)-arm64 \
+        GO_BASE_IMAGE=$(GO_BASE_IMAGE) \
+        ALPINE_BASE_IMAGE=$(ALPINE_BASE_IMAGE) \
```

Comment:
could you keep it consistent to use `/t` instead of spaces here? (Makefile sometimes is strict on using tabs.

Reply:
Good catch. Copy and paste in the vi editor changed the /t to spaces. Fixed.